### PR TITLE
[WIP] Expose the new vsphere driver parameters

### PR DESCRIPTION
**Pull request by codyrancher**
Exposed:
    1. Creation type
    2. Clone from
    3. Cloud init type
    
    And modified the following input for improved UX:
    1. Cloud init is now a textarea instead of a text field.
    2. Network is now a form-value-array

rancher/rancher#21580

Highlighting the changes to Instance Options:
Screen Shot 2019-09-12 at 4 53 51 PM

The available creation type options:
Screen Shot 2019-09-12 at 4 53 59 PM

The available cloud init type options:
Screen Shot 2019-09-12 at 4 54 05 PM

Highlighting changes to Scheduling Options:
Screen Shot 2019-09-12 at 4 54 37 PM

### CODE REVIEW

File: lib/nodes/addon/components/driver-vmwarevsphere/template.hbs [hunk 1]
```diff
@@ -84,36 +84,59 @@
...
+            class="form-control"
+            content=cloudinitTypeOptions
+            value=config.cloudinitType
+        }}
+      </div>
+      <div class="col span-6">
+        <label class="acc-label">
+          {{t "nodeDriver.vmwarevsphere.cloudinit.label"}}
+        </label>
+        {{textarea
```
[the remark this answers is not in the document]
Comment (codyrancher) [reply lib/nodes/addon/components/driver-vmwarevsphere/template.hbs]: Done

**Comment by cjellick** [comment]
Note: this change is not going to land in v2.3.0, its scheduled/sotted for, like v2.3.2. So, @vincent99 you might want to hold this one before merging or cut a release branch for v2.3 (causing more backport work :-/ )

**Comment by cjellick** [comment]
@codyrancher as with the other issue, drop "resolves" from the description so that it doesnt auto-close the issue

**Comment by codyrancher** [comment]
> @codyrancher as with the other issue, drop "resolves" from the description so that it doesnt auto-close the issue

Done.

**Comment by vincent99** [comment]
Ok I'll just mark it WIP for now so it stands out not to merge.

**Pull request closed without merging**